goawk.go

```diff
@@ -337,6 +398,18 @@ argsLoop:
 	os.Exit(status)
 }
 
+func coverModeFromString(mode string) cover.Mode {
+	switch mode {
+	case "set":
+		return cover.ModeSet
+	case "count":
+		return cover.ModeCount
+	default:
+		errorExitf("covermode can only be one of: set, count")
```

Comment:
Let's tweak this to include the dash: `"-covermode can only be one of: set, count"`